src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -0,0 +1,160 @@
+    <Collapser
+    id="step1"
+    title="Step 1. Clone the Snowflake repository"
+    >
+    Run the following command in your CLI to clone the Snowflake repo into your machine:
+
+    ```
+    git clone https://github.com/newrelic/newrelic-snowflake-integration.git
+    ```
+    </Collapser>
+    <Collapser
+    id="step2"
+    title="Step 2. Download relevant binary for your platform"
+    >
+In the cloned repo from step 1, download the [relevant binary](https://github.com/newrelic/newrelic-snowflake-integration/releases) in the `releases` folder. In other words, if you're running Linxus, then you'd download `snowflakeintegration-linux.` 
+
+Be sure the binary is executable for your platform. In other words, run the following command:
+
+* For Linux, run `chmod +x snowflakeintegration-linux`
+* For Windows, run `chmod +x snowflakeintegration-win.exe`
+* For MacOS, run `chmod +x snowflakeintegration-macos`
+
+    </Collapser>
+    <Collapser
+    id="step3"
+    title="Step 3. Update `config.yaml`"
+    >
+To ensure that you're using the most up to date information for the integration, update your `config.yaml` file to match the snippet found in [Key Pair authentication](https://github.com/newrelic/newrelic-snowflake-integration#key-pair-authentication) in the Snowflake repo. 
+
+Once you've updated the yaml file, return to this doc to install the dashboards to your New Relic account. 
+    </Collapser>
+</CollapserGroup>
+
+## Create custom queries [#custom]
+
+`TEXT HERE PENDING PHANEE'S RESPONSE` 
+
+<CollapserGroup>
+    <Collapser
+        className="freq-link"
+        id="longest"
+        title="Longest queries"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `longest_queries.sql` 
+    2. Designate the file path as `newrelic-snowflake-integrations/queries`.
+    3. In `longest_queries.sql`, add the following snippet:
+
+    ```sql
+    select 
+        query_id, 
+        query_text,
+        (execution_time / 60000) as exec_time,
+        warehouse_name,
+        user_name,
+        execution_status
+    from 
+        snowflake.account_usage.query_history 
+    where 
+        execution_status = 'SUCCESS' 
+    order by 
+        execution_time desc
+    ```
+    4. In the `flex-snowflake.yml` file, add the following snippet:
+    
+```yml
+- name: longestQueries
+entity: snowflake
+event_type: SnowflakeVirtualWarehouse
+custom_attributes:
+    metric_type: snowflake.query_performance
+commands: 
+    - run: <var>YOUR_PATH_TO snowflakeintegration-platform</var> <var>YOUR_PATH_TO config.yaml</var> <var>YOUR_PATH_TO longest_queries.sql</var>
+```
+    </Collapser>
+    <Collapser
+        className="freq-link"
+        id="warehouse-usage"
+        title="Usage by warehouse"
+    >
+    To find the queries that take the longest to load:
+
+    1. Create a file and name it `credit_usage_by_warehouse.sql` 
+    2. Designate the file path as `newrelic-snowflake-integrations/queries`.
```

Comment:
2. Designate the file path as YOUR_PATH_TO_CLONED_REPOSITORY_DIRECTORY/queries/credit_usage_by_warehouse.sql